### ISSUE TITLE
intermittent delay occurs in certain regions

### ISSUE THREAD

**Issue by FreshCreamBun** [aws/aws-sdk-js-v3#3216]
## Describe the bug
Hello, I am a developer working for an AWS Partner. I've been using javascript v3 aws-sdk to get information about customers' use of AWS services.

I'm experiencing intermittent communication delays when requesting a URL for the 'eu-west-1' region, which is the Ireland region among several regions.

The region making the AWS API request is Seoul, South Korea, and several services in other regions are communicating smoothly.

Communication delays to the 'eu-west-1' region are critical, as customers must periodically update their usage of multiple services.

Please help me troubleshoot the 'eu-west-1' region communication issue.

#### SDK version number
`@aws-sdk/client-ec2@3.33.0`

#### Is the issue in the browser/Node.js/ReactNative?
Node.js(NestJS)

#### Details of the browser/Node.js/ReactNative version
Node.js 

## Steps to reproduce
ex) code
```javascript
const ec2Client = new EC2Client({
                            region: ec2Region,
                            credentials: awsCredentials,
                            tls: false,
        })
        // EC2
        ec2Client.send(new DescribeInstancesCommand({}))
```
ex) return
```
{
  '$metadata': {
    httpStatusCode: 200,
    requestId: '*******',
    extendedRequestId: undefined,
    cfId: undefined,
    attempts: 3,
    totalRetryDelay: 508
  },
  Reservations: [],
  NextToken: undefined
}
```

**Comment by vudh1** [comment]
Hi @kcm5756 I cannot reproduce this behavior. Can you confirm if this still persisting or a network problem?

**Comment by kcm5756** [comment]
> Hi @kcm5756 I cannot reproduce this behavior. Can you confirm if this still persisting or a network problem?

Sorry for the late confirmation of the issue.

After posting the issue, we found out that the problem occurred after our company replaced the firewall equipment,    
and now we have moved the server to a network farm that uses another firewall equipment.
As a result, all AWS API communication, including the 'eu-west-1' region, is running smoothly.

It is difficult to determine the exact cause,     
but it is presumed that there was a communication problem with the 'eu-west-1' region due to a firewall equipment problem.
